pkg/util/util_test.go

```diff
@@ -31,13 +30,13 @@ var _ = Describe("Util", func() {
 		Expect(result).To(Equal(expectedResult))
 	},
 		table.Entry("Valid namespace", filepath.Join(fileDir, "namespace.txt"), "test-namespace"),
-		table.Entry("Invalid file", "doesnotexist", v1.NamespaceSystem),
+		table.Entry("Invalid file", "doesnotexist", "cdi"),
```

Comment:
Do we have a constant for the testing deployment namespace?

Reply:
nope